proto/viam/app/v1/app.proto

```diff
@@ -255,6 +255,21 @@ message RobotPartHistoryEntry {
   string robot = 2 [(tagger.v1.tags) = "bson:\"robot\" json:\"robot\""];
   google.protobuf.Timestamp when = 3 [(tagger.v1.tags) = "bson:\"when\" json:\"when\""];
   RobotPart old = 4 [(tagger.v1.tags) = "bson:\"old\" json:\"old\""];
+  AuthenticatorInfo who = 5 [(tagger.v1.tags) = "bson:\"who\" json:\"who\""];
```

Comment:
I'm thinking we should mark this as optional - if the caller doesn't have permission to see who edited, this could just be empty. Thoughts ?